templates/docs/patterns/slider.md

```diff
@@ -15,7 +15,9 @@ when you want to provide a numeric representation of the slider value, as well a
 View example of the slider pattern
 </a></div>
 
-<span class="p-label--updated">Updated</span> In version 2.6.0 Vanilla framework added slider styling as default for
+<div class="p-chip--information is-inline is-dense">
```

Comment:
I know our examples show them as divs (which is wrong I think), but we should use spans here. Otherwise divs break out of paragraphs (because you can't put `<div>` in `<p>`):

<img width="708" alt="Screenshot 2021-11-26 at 17 06 58" src="https://user-images.githubusercontent.com/83575/143607436-4cfb4475-1fb3-43fb-a8c9-805718db4906.png">


```suggestion
<span class="p-chip--information is-inline is-dense">
```